addons/website_version/models/ir_ui_view.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+from lxml import etree
+from openerp import tools, fields, models, api
+
+
+class view(models.Model):
+
+    _inherit = "ir.ui.view"
+
+    version_id = fields.Many2one('website_version.version', ondelete='cascade', string="version_id")
+
+    @api.multi
+    def write(self, vals, toggle=False):
```

Comment:
If it is linked to a change of active, could'nt you compute it ? Like toggle = vals.get('active') and all(self.active != vals.get('active')) or False ?